jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -112,6 +112,18 @@
           ></v-select>
         </v-row>
 
+        <v-row v-if="current_plot_type==='Radial Profile (Raw)' && subset_area > 5000">
+          <span class="v-messages v-messages__message text--secondary">
+              <b>WARNING</b>: Computing and displaying raw profile of an aperture containing ~{{subset_area}} pixels may be slow or unresponsive.
+          </span>
+        </v-row>
+        <v-row v-if="current_plot_type==='Radial Profile (Raw)'">
+          <span class="v-messages v-messages__message text--secondary">
+              Estimated remaining memory usage: {{predictedMemoryUsagePerc(subset_area).toPrecision(2)}}%

```

Comment:
```suggestion
              Estimated remaining memory: {{predictedMemoryUsagePerc(subset_area).toPrecision(2)}}%
```

Would absolute number be more meaningful? For example: `100 / 500 MB` (if that is indeed in MB)

Reply:
Probably, but this is just for internal testing (at least for now).  If we decide to expose this information in the warning, we should think about how best to represent it.

Reply:
(moved to #1821)